byway/src/main/webapp/Destinations.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+    <script
+      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAljWlTjyagvHyZTN69Mvun2mH4Eq8j9yI&callback=initAutocomplete&libraries=places&directions&v=weekly"
+      defer
+    ></script>
+    <link href="https://fonts.googleapis.com/css2?family=Karla&family=Rubik:wght@500;700&display=swap" rel="stylesheet">
+  </head>
+  <body id="destinations-page" >
+
+    <div class="header">
+      <img src="/images/logo.png" alt="logo" class="logo">
+      <div id="name">byway.</div>
+    </div>
+    
+      
+    <div id="progressbar-container">
+      <ul id="progressbar">
+          <li>Choose Destinations</li>
+          <li>choose interests</li>
+          <li>View route</li>
+        </ul>
+    </div>
+    
+    <div id="map-search-container">
+        <div id="map"></div>
+        <div id= "destination-input-form">
+        <form action="/destinations" method="POST">
+
+            <input
+                id="pac-input"
```

Comment:
Choose meaningful names for element IDs. Note that if your input names are unique per page, it's not a bad idea to also use them as form input ids.

Reply:
changed the ids